protocol.cpp

```diff
@@ -706,17 +706,22 @@ int redis_protocol::write_arbitrary_command(const char *rand_val, int rand_val_l
 }
 
 bool redis_protocol::format_arbitrary_command(arbitrary_command &cmd) {
+    int key_placeholder_count = 0;
     for (unsigned int i = 0; i < cmd.command_args.size(); i++) {
         command_arg* current_arg = &cmd.command_args[i];
         current_arg->type = const_type;
 
         // check arg type
         if (current_arg->data.find(KEY_PLACEHOLDER) != std::string::npos) {
+            key_placeholder_count++;
             if (current_arg->data.length() != strlen(KEY_PLACEHOLDER)) {
                 benchmark_error_log("error: key placeholder can't combined with other data\n");
                 return false;
             }
-
+            if (key_placeholder_count > 1) {
```

Comment:
This should apply only to cluster mode. The non-cluster mode does support more than one key.

Reply:
Also, _"IMOH just dropping keys is not the right solution and if yes, maybe only for this case of arbitrary command in Cluster mode."_ was relevant for the initial PR where we allowed more than one key.
Since we agree to limited the arbitrary command in cluster mode for one key (for now) there is no need to "play" with the pool

Reply:
> This should apply only to cluster mode. The non-cluster mode does support more than one key.

@YaacovHazan both for standalone and cluster versions we don't allow for more than one key placeholder.